src/index.js

```diff
@@ -226,10 +223,21 @@ class Card extends React.Component {
   checkWindowPosition(style, arrowStyle) {
     if (this.props.position === 'top' || this.props.position === 'bottom') {
       if (style.left < 0) {
-        let offset = style.left
+        const parent = this.props.parentEl
+        if (parent) {          
+          const tooltipWidth = this.state.width
+          let bgStyleRight = arrowStyle.bgStyle.right
+          // For arrow = center
+          if (!bgStyleRight) {
+            bgStyleRight = (tooltipWidth / 2) - BG_SIZE
+          }
+          const newBgRight = Math.round(bgStyleRight - style.left + this.margin)
+          arrowStyle = Object.assign({}, arrowStyle, {
```

Comment:
could use `assign` from `object-assign` for consistency

Reply:
Done 👍

Reply:
Thanks!